contentctl/objects/dashboard.py

```diff
@@ -0,0 +1,96 @@
+from typing import Any
+from pydantic import Field, Json, model_validator
+
+import pathlib
+import copy
+from jinja2 import Environment
+import json
+from contentctl.objects.security_content_object import SecurityContentObject
+from contentctl.objects.config import build
+
+DEFAULT_DASHBAORD_JINJA2_TEMPLATE = '''<dashboard version="2" theme="light">
+    <label>{{ dashboard.label(config) }}</label>
+    <description></description>
+    <definition><![CDATA[
+{{ dashboard.pretty_print_json_obj() }}
+    ]]></definition>
+    <meta type="hiddenElements"><![CDATA[
+{
+	"hideEdit": false,
+	"hideOpenInSearch": false,
+	"hideExport": false
+}
+    ]]></meta>
+</dashboard>'''
+
+class Dashboard(SecurityContentObject):
+    j2_template: str = Field(default=DEFAULT_DASHBAORD_JINJA2_TEMPLATE, description="Jinja2 Template used to construct the dashboard")
+    description: str = Field(...,description="A description of the dashboard. This does not have to match "
+                             "the description of the dashboard in the JSON file.", max_length=10000)
+
+    json_obj: Json[dict[str,Any]] = Field(..., description="Valid JSON object that describes the dashboard")
```

Comment:
I originally though about trying to have this JSON object live in the YML, but it resulted in an ugly and long YML.
The contents of this are parsed into the object at creation time.

Reply:
LGTM